style.css

```diff
@@ -0,0 +1,6 @@
+main {
+  display: block;
+  margin: auto;
+  max-width: 100%;
+  height: auto;
+}
```

Comment:
Please try adding more styling to the form to make it more attractive. (Like background images, font styles, font colours etc.)  🌈💫

Reply:
[OPTIONAL] I suggest you show a copy of the original app in freecodecamp for comparison. Its always great to create a clear picture to collaborator.